MediaBrowser.Model/Lyrics/TimeTag.cs

```diff
@@ -0,0 +1,28 @@
+namespace MediaBrowser.Model.Lyrics;
+
+/// <summary>
+/// TimeTag model.
+/// </summary>
+public class TimeTag
+{
+    /// <summary>
+    /// Initializes a new instance of the <see cref="TimeTag"/> class.
+    /// </summary>
+    /// <param name="position">The character index of the lyric.</param>
+    /// <param name="timestamp">The timestamp the lyric is synced to in ticks.</param>
+    public TimeTag(int position, long timestamp)
```

Comment:
The lyric line uses "Start" so why do we use "Timestamp" here? That's not really consistent